devito/builtins.py

```diff
@@ -123,46 +135,61 @@ def fset(f, g):
         else:
             raise NotImplementedError
 
-    lw = int(_order*sigma + 0.5)
+    try:
+        dtype = f.dtype.type
+    except AttributeError:
+        dtype = f.dtype
+
+    # TODO: Add s = 0 dim skip option
+    lw = tuple([int(_truncate*float(s) + 0.5) for s in as_tuple(sigma)])
```

Comment:
nitpicking, pythonicity:
no need for the square brackets 
`int(_truncate*float(s) + 0.5) for s in as_tuple(sigma)` creates a generator, and `tuple(...)` expands it.

Reply:
Ta.